### PR TITLE
Remove non-Garnett rich link markup

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Are rich links ever going to appear again in non-Garnett mode? You can bet that they won't. This change removes the markup specific to non-Garnett rich links.

## What is the value of this and can you measure success?

Less old code

### CODE REVIEW

File: onward/app/views/fragments/richLinkBody.scala.html [hunk 1]
```diff
@@ -43,23 +42,11 @@
...
-                        (content.content.cardStyle == SpecialReport && content.content.hasTonalHeaderByline)
-                    ) {
-                        @fragments.inlineSvg("quote", "icon")
-                    }
-                } else {
-                    @if(content.content.cardStyle == Comment) {
-                        @fragments.inlineSvg("garnett-quote", "icon")
-                    }
+                @if(content.content.cardStyle == Comment) {
+                    @fragments.inlineSvg("garnett-quote", "icon")
```
Comment (SiAdcock) [inline onward/app/views/fragments/richLinkBody.scala.html]: Perhaps we need a better name for this icon? The word `garnett` will probably be meaningless in 6 months.

File: onward/app/views/fragments/richLinkBody.scala.html [hunk 2]
```diff
@@ -78,41 +65,19 @@ <h1 class="rich-link__title">
             </div>
         }
 
-        @if(!experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
-            @if(!content.tags.isGallery && !content.tags.isVideo) {
-                <div class="rich-link__standfirst u-cf">
```
Comment (SiAdcock) [inline onward/app/views/fragments/richLinkBody.scala.html]: `rich-link__standfirst` and later `rich-link__byline-img` and `rich-link__byline-img__img`: these seem to be the only occurrences of these classes in the templates... perhaps we can also delete the corresponding CSS rules from the stylesheets?

File: onward/app/views/fragments/richLinkBody.scala.html (hunk 1 above)
Comment (GHaberis) [reply onward/app/views/fragments/richLinkBody.scala.html]: Is the old `quote.svg` used anywhere still? Could remove the old asset and rename `garnett-quote.svg` to `quote.svg` and?

File: onward/app/views/fragments/richLinkBody.scala.html (hunk 2 above)
Comment (GHaberis) [reply onward/app/views/fragments/richLinkBody.scala.html]: If you're happy to do this... 👍

**Review by GHaberis — APPROVED** [review]

File: onward/app/views/fragments/richLinkBody.scala.html (hunk 1 above)
Comment (paperboyo) [reply onward/app/views/fragments/richLinkBody.scala.html]: On the Opinion front?
old quote

Comment (SiAdcock) [reply onward/app/views/fragments/richLinkBody.scala.html]: Yeah, it's still used in several places. I don't think it's ready to remove quite yet

**Pull request merged**